crates/pipeline_manager/src/compiler.rs

```diff
@@ -329,7 +343,8 @@ impl Compiler {
 
         // `cargo build`.
         let mut cargo_process =
-            Compiler::run_cargo_build(config, program_id, &CompilationProfile::Optimized).await?;
```

Comment:
Does optimized here help with unoptimized default builds? I just kept the current value.

Reply:
@ryzhyk ?

Reply:
Hmm, in a next compiler rework PR we might want precompilation to be done for all profiles